ceph-windows-test/build/run_tests

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o pipefail
+
+CEPH_WIN_BIN_DISTRO=${CEPH_WIN_BIN_DISTRO:-"windows"}
+CEPH_WIN_BIN_DISTRO_VERSION=${CEPH_WIN_BIN_DISTRO_VERSION:-"1809"}
+CEPH_WIN_BIN_SHA1=${CEPH_WIN_BIN_SHA1:-"c7f2056f7415c5d26b0503e7c13a89e6b4856341"}
+
+VM_IMAGE_URL=${VM_IMAGE_URL:-"https://filedump.ceph.com/windows/ceph-win-ltsc2019-ci-image.qcow2"}
+VM_IMAGES_DIR=${VM_IMAGES_DIR:-"$HOME/vm_images"}
+
+MON_IP=${MON_IP:-"192.168.122.1"}
+CEPHADM_RELEASE=${CEPHADM_RELEASE:-"pacific"}
```

Comment:
We are not building Ceph here. We're just downloading artifacts from Chacra with the Windows build, and run the unit tests on a Windows VM.

The unit tests need a running Ceph cluster. And we use `cephadm` to spawn a minimal cluster for the tests, which is cleaned up at the end of the job.

I didn't want the `cephadm` release to be hard-coded in the script, that's why I added a variable for it, with the default value being the most recent Ceph release.

Reply:
I understand, but, if there are changes in Ceph that require changes in cephadm...  Granted, you're a little isolated in that you're not running *daemons* on Windows, but one can easily imagine, say, a new required config option.

I guess it's okay, but desynchronization still seems remotely possible.

Reply:
@dmick I'm not sure I understand your concern.

Do you believe that there's no value in keeping a variable for the `cephadm` to be downloaded, and used to bootstrap a testing cluster ?

Reply:
I'm nervous about some Ceph core code/library change making the job fail because it requires a cephadm change of some sort (bug fix, perhaps).  Master Ceph should always work with master cephadm, said another way.

It's probably pretty low-risk.

Reply:
I see. To be honest, I can make the job use master `cephadm` all the time.

In fact, I intended to do that. But I didn't find a known location to download `cephadm` from master builds (which I know we have).

Do you know if we have a way to download latest `cephadm` that would always correspond to our latest master build ?

Reply:
at the moment, it's just one script file that could come from the source tree.  At some point it may change to a .zip file, so would need to be the output of a build; it's stored in the build repo output as a plain file.  But neither of those are particularly easy to find at the moment.  

Let's just let this be for now, and if it turns out to be a maintenance issue, we'll address it in future.  Other jobs have this same issue.

Reply:
Sounds good.